walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -8,12 +8,28 @@
   <body>
     <div id="content">
       <h1>Laura's Portfolio</h1>
-      <p>This is Laura, I am a rising junior at MIT. I am currently majoring in Computer Science. 
-	I was born and raised in Venezuela and moved to the US to pursue higher education.</p>
+      <p>My name is Laura Queipo, I am a rising junior at MIT. I am currently majoring in Computer 
+	Science and Engineering and planning to minor in Brain and Cognitive Science. On campus, I am
+        mainly involved with the international community. Indeed, I am a member of the International
```

Comment:
Not an issue but noting that it's weird that it's indented here (I don't know if that's your formatting or github's)

Reply:
Ohh you mean the line starting with mainly?

Reply:
Yeah that one

Reply:
I think it might be github's when I see the file I can't see the indentention